fastlane/lanes/localization.rb

```diff
@@ -180,7 +177,6 @@
       skip_upload_apk: true,
       skip_upload_aab: true,
       skip_upload_metadata: false,
-      # TODO: enable changelogs upload once we drop the `android_create_xml_release_notes` action call (and drop the step in release scenario)
       skip_upload_changelogs: true,
```

Comment:
Removed that TODO without actually enabling `skip_upload_changelogs: false` here like it initially suggesteed after all, because I realized that this TODO that I wrote back then… was not correct.

Indeed, this `upload_playstore_localized_metadata ` lane is still intended to only update the metadata (as in: app title, short description, long description, …), and for production builds only (hence the `track: 'production'` on L177), and not related to the changelogs (which are uploaded during a different lane, namely `upload_build_to_play_store`, at the same time we upload the `apk`/`aab` after a build)